### ISSUE TITLE
Error showing in jupyter notebook

### ISSUE THREAD

**Issue by ShakhayatHossainShykat** [matterport/Mask_RCNN#2604]
![image](https://user-images.githubusercontent.com/44720412/122441509-6d09e400-cfbf-11eb-9644-f0a5a46e0d82.png)
Need a solution for this error. Please help!

**Comment by salma-achour** [comment]
You just need to install Tensorflow in your environment.
Here's a guide on how to do it [Install TensorFlow with pip](https://www.tensorflow.org/install/pip)
If you're using a conda environment you can follow this guide [conda install tensorflow](https://anaconda.org/conda-forge/tensorflow)

PS: If you have a GPU, you can install Tensorflow's GPU version `tensorflow-gpu` .

**Comment by ShakhayatHossainShykat** [comment]
Yes, Thank you I installed all the required dependencies. Now I am having this error massage.
![image](https://user-images.githubusercontent.com/44720412/122583030-9fc4e280-d07a-11eb-911e-c019c3b655dc.png)

**Comment by MahejabeenNidhi** [comment]
Mask RCNN is compatible with older versions of tensorflow and keras. I had to downgrade further to tensorflow 1.13.1, it worked but a lot of the variables were depreciated. Other needed packages are likely 
h5py==2.10.0

Mask R-CNN demo is trained on MS COCO dataset. COCO is a large-scale object detection, segmentation, and captioning dataset. 
To install pycocotools on Windows:
Conda install git
Pip install Cython
pip3 install "git+https://github.com/philferriere/cocoapi.git#egg=pycocotools&subdirectory=PythonAPI"
